intermediate_source/fx_conv_bn_fuser.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+(beta) Building a Conv/BN fuser in FX
+*******************************************************
+**Author**: `Horace He <https://github.com/chillee>`_
+
+In this tutorial, we are going to use FX to do the following:
+
+1) Find patterns of conv/batch norm in the data dependencies.
+2) Fold the batch norm statistics into the convolution weights.
+
+Note that this optimization only works for models in inference mode.
+
+We will be building the fuser that exists here:
+https://github.com/pytorch/pytorch/blob/master/torch/fx/experimental/fuser.py
```

Comment:
I usually recommend linking to an explicit version (could be the 1.8 branch) because this will go out of date pretty quickly